pkg/deploy/registry/replication/etcd/etcd.go

```diff
@@ -0,0 +1,69 @@
+package etcd
```

Comment:
This file will go away as we only need to plug the strategy in the upstream replication controller storage.